File: PHP/Insertionsort.php
<?php
function Insertionsort($arr,$size){
    for($i=1;$i<$size;$i++){
        $min=$arr[$i];
        $j=$i-1;
        while ($j>=0 && $arr[$j]>$min) {
            $arr[$j+1]=$arr[$j];
            $j--;
        }
        $arr[$j+1]=$min;
    }
    print_r($arr);
}

$size=(int)readline('Enter Size of an array= > ');
$arr=[];
for($i=0;$i<$size;$i++){
    $arr[$i]=(int)readline();
}
Insertionsort($arr,$size);
?>

File: Java/Insertionsort.java
import java.util.Scanner;
import java.util.Arrays;
class Insertionsort{
    static void InsertionSort(int arr[],int size){
        for(int i=1;i<size;i++){
            int min=arr[i];
            int j=i-1;
            while (j>=0 && arr[j]> min) {
                arr[j+1]=arr[j];
                j--;
            }
            arr[j+1]=min;
        }
        System.out.println("Sorted Array =>"+Arrays.toString(arr));
    }
public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    System.out.print("Enter The Size of array => ");
    int size = sc.nextInt();
    int[] arr=new int[size];
    for(int i=0;i<size;i++){
        arr[i]=sc.nextInt();
    }
    sc.close();
    InsertionSort(arr, size);
}
}